tests/TestRvsdgs.hpp

```diff
@@ -855,6 +855,141 @@ class GammaTest final : public RvsdgTest {
 	jlm::rvsdg::gamma_node * gamma;
 };
 
+/** \brief GammaTest2 class
+ *
+ * This function sets up an RVSDG representing the following function:
```

Comment:
It's a class and no specific function is mentioned.

Reply:
Fixed.